### PR TITLE
Oc 320 oc 319 map view

### PR DESCRIPTION
**Pull request by Dan-Wood**

### CODE REVIEW

File: src/components/admin/utils/DraggableMap.tsx [hunk 1]
```diff
@@ -66,7 +66,6 @@ export default class DraggableMap extends React.Component<Props, State> {
     this._isMounted = true;
     this.initialiseMap();
     if (this.map) {
-      this.locateUser();
```
Comment (matthewberryman) [inline src/components/admin/utils/DraggableMap.tsx]: Is this component re-used? We still want this.locateUser() if there's no geospatial metadata for an item/collection (i.e. if it's new, or we are editing an existing item with nothing added). @Dan-Wood

File: src/components/admin/utils/DraggableMap.tsx [hunk 2]
```diff
@@ -345,29 +353,6 @@ export default class DraggableMap extends React.Component<Props, State> {
     }
   }
 
-  /**
```
Comment (matthewberryman) [inline src/components/admin/utils/DraggableMap.tsx]: ditto previous comment

File: src/components/admin/utils/DraggableMap.tsx (hunk 1 above)
Comment (Dan-Wood) [reply src/components/admin/utils/DraggableMap.tsx]: Yes you're right.

Comment (matthewberryman) [reply src/components/admin/utils/DraggableMap.tsx]: in fact I get an error thrown in some cases

Comment (matthewberryman) [reply src/components/admin/utils/DraggableMap.tsx]: ×
Error: Bounds are not valid.
NewClass.fitBounds
node_modules/leaflet/dist/leaflet-src.js:3062
NewClass.addData
src/components/admin/utils/DraggableMap.tsx:168

Comment (Dan-Wood) [reply src/components/admin/utils/DraggableMap.tsx]: Yep on that now.

**Comment by Dan-Wood** [comment]
> I'm still not really happy with the "fix" for map autofocus on pan hiding the dropzone and other things.
> Can we not just set the size of the map so all of it is shown in the space for it in the model, whatever size the modal happens to be? This also fixes the issue where we zoom to bounds (for items where existing geospatial metadata is present) but not all of the geoms are shown because the map is being cut off.

There has to be a minimum height though, the controls will get cut off (that height is 360px), you're not particularly going to get around this completely.

Another option is the upload and inputs on the left or right and the map sitting next to it.

**Pull request merged**